.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Prime Data Hub
+
+on:
+  push:
+    branches: [ master, production ]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set Environment Variables
+        run: |
+               if [ "$GITHUB_BASE_REF" == "production" ]
+               then
+                   echo "Deploying to the production environment."
+                   echo >> $GITHUB_ENV RESOURCE_GROUP=prime-data-hub-prod
+                   echo >> $GITHUB_ENV AZ_FXNS_DEFAULT_KEY=${{ secrets.AZ_FXNS_PROD_DEFAULT_KEY }}
+                   echo >> $GITHUB_ENV POSTGRESQL_USER=${{ secrets.POSTGRESQL_PROD_USER }}
+                   echo >> $GITHUB_ENV POSTGRESQL_PWD=${{ secrets.POSTGRESQL_PROD_PWD }}
+                   echo >> $GITHUB_ENV SFTP_CREDS=${{ secrets.SFTP_PROD_CREDS }}
+               else
+                   echo "Deploying to the test environment."
+                   echo >> $GITHUB_ENV RESOURCE_GROUP=prime-data-hub-test
+                   echo >> $GITHUB_ENV AZ_FXNS_DEFAULT_KEY=${{ secrets.AZ_FXNS_TEST_DEFAULT_KEY }}
+                   echo >> $GITHUB_ENV POSTGRESQL_USER=${{ secrets.POSTGRESQL_TEST_USER }}
+                   echo >> $GITHUB_ENV POSTGRESQL_PWD=${{ secrets.POSTGRESQL_TEST_PWD }}
+                   echo >> $GITHUB_ENV SFTP_CREDS=${{ secrets.SFTP_TEST_CREDS }}
+               fi
+               env
+
+      - name: Login to Azure CLI
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.SERVICE_PRINCIPAL_CREDS }}
+
+      - name: Update Azure Functions App
+        run: az functionapp update --resource-group ${{ env.RESOURCE_GROUP }} --name ${{ env.RESOURCE_GROUP }}
```

Comment:
Based on the documentation and my observations in the activity log, this will trigger the app to update and pull in the new image.  We would no longer need the webhook to do the CD process and it can all be self-contained within GitHub.